packages/@react-aria/focus/src/useFocusable.tsx

```diff
@@ -76,7 +76,7 @@ export function useFocusable(props: FocusableOptions, domRef: RefObject<HTMLElem
       domRef.current.focus();
     }
     autoFocusRef.current = false;
-  }, []);
+  }, [domRef]);
```

Comment:
why was this added? when does it change other than mount? or is this just lint cleanup?

Reply:
lint cleanup. I'll replace it with a comment.